contracts/airdrops/Existing1155NftDrop.sol

```diff
@@ -139,40 +101,24 @@ contract Existing1155NftDrop is AirdropInfo, AirdropMerkleProof, IERC1155Receive
         rewardToken.safeTransferFrom(address(this), msg.sender, rewardTokenId, tokensPerClaim * tokenIds.length, "");
     }
 
-    //@notice Get the type of airdrop, it's either ERC20, ERC721, ERC1155
-    function getAirdropType() external pure override returns (string memory) {
-        return "ERC1155";
-    }
-
-    //@notice Checks if the user is eligible for this airdrop
-    function isEligibleForReward(uint256 tokenId) external view returns (bool) {
+    /// @notice Checks if the user is eligible for this airdrop
+    function isEligibleForReward(uint256 tokenId) public view returns (bool) {
```

Comment:
natspec @return missing

Reply:
Just to check, would this be appropriate?
<img width="848" alt="image" src="https://user-images.githubusercontent.com/56925545/180454769-aab124ed-3a45-4fca-a3fe-e317cec9e3a8.png">

Since

Reply:
Also, we have not added the natspec for return value anywhere in this project so we'd have to fix that too

Reply:
You can do it like this, or `@return true if user is eligible for reward` or something like that. 
It is a good practice to see well-documented code, unless you are in a hurry to deploy, I'd make it right

Reply:
I'd change naming a bit for this, or the way it is written.

The function is called `isEligibleForReward` and returns a bool
Without looking at the code, and just looking at this line `function isEligibleForReward(uint256 tokenId) public view returns (bool)`
I'd expect this function to return true/false
Because this function never returns false, it wastes gas by returning `true`.

In our case we never return `false`, instead we revert for some reason.
If you want to keep the code this way I recommend renaming to something like `checkRewardEligibility` or `validateRewardEligibility`, or something like that.
Imagine that you're looking at the code that somebody wrote and you see `isOver18`, You wouldn't expect a revert if somebody is 15, `false` makes more sense.

This is not a major issue, just the way I think when i read function signatures.

Reply:
Fixed the natspec's and addid a validation function, whilst making sure the isEligibleForReward function only returns true or false.